src/admin-views/payments/tickets-commerce.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * The Template for displaying the Tickets Commerce Payments Settings.
+ *
+ * @version TBD
+ */
+
+use TEC\Tickets\Commerce\Gateways\PayPal\Merchant;
+use TEC\Tickets\Commerce\Gateways\PayPal\Signup;
+
+$merchant = tribe( Merchant::class );
+$is_merchant_active = $merchant->is_active();
+
+$display = '<div class="tec-tickets-commerce-paypal-connect">';
+
+if ( $is_merchant_active ) {
+	$disconnect_url = Tribe__Settings::instance()->get_url( [ 'tab' => 'payments', 'tc-action' => 'paypal-disconnect' ] );
+	$name = $merchant->get_merchant_id();
+
+	$disconnect = ' <a href="' . esc_url( $disconnect_url ) . '">' . esc_html__( 'Disconnect', 'event-tickets' ) . '</a>';
+	$display .= '<p>' . esc_html__( 'PayPal Status: Connected', 'event-tickets' ) . '</p>';
+	$display .= '<p>' . esc_html( sprintf( __( 'Connected as: %1$s', 'event-tickets' ), $name ) ) . $disconnect . '</p>';
+} else {
+	$display .= '<h2>' . esc_html__( 'Accept online payments with PayPal!', 'event-tickets' ) . '</h2>
+				' . esc_html__( 'Start selling tickets to your events today with PayPal. Attendees can purchase tickets directly on your site using debt or credit cards with no additional fees.',
+			'event-tickets' ) . tribe( Signup::class )->get_link_html();
```

Comment:
formatting? also, should the text be surrounded by `<p>` tags?

Reply:
Will be addressed by Juan, adding a todo around this whole file.